src/transformers/models/big_bird/modeling_big_bird.py

```diff
@@ -1397,6 +1398,9 @@ def forward(
                 hidden_states, band_mask, from_mask, to_mask, from_blocked_mask, to_blocked_mask, output_attentions
             )
 
+        self_outputs = tuple(
```

Comment:
@xuzhao9 - thanks a lot for the fix. Instead of converting all tensors here to the dtype of `hidden_states` in every layer at every forward pass, could we try to only do where it's necessary? E.g. is it maybe enough to just correctly cast the `band_mask`, `from_mask` and `to_mask` once before forwarding them into `self.self(...)`?

Reply:
@patrickvonplaten Sorry for the late response. I have addressed your comments. Can you help take another look?